CHANGELOG.md

```diff
@@ -2,8 +2,88 @@
+- Platform: Custom attributes support in reporting and expressions
+  - Now selectable as a column
+  - Support in report filters and charts
+- Providers
+  - Amazon: Public Images Filter
+  - Networking
+      - Separate Google Network Manager
+      - NFV: VNFD Templates and VNF Stacks
+  - Hawkular
+    - Deployment entity operations: Undeploy, redeploy
+    - Server operations: Reload, suspend, resume
+    - Live metrics for datasources and transactions
+    - Performance reports for middleware servers
+    - Support for alert profiles and alert automated expressions for middleware server
+    - Crosslink middleware servers with RHEV VMs
+    - Collect and display deployment status
+    - Datasources topology view
+- REST API
+  - Support for report schedules
+  - Support for approving or denying service requests
+  - Support for OpenShift Container Deployments
+  - Support for Virtual Templates
+- Service Broker
+  - Started work on Service Broker to allow ManageIQ to select VM for you based on criteria (cloud, cost, or performance)
+  - Added API backend for Resourceless Servers
+  - Added datastore for the default settings for resourceless servers
+- SmartState: Generalized disk LRU caching module
+  - Caching module can be used by any disk module, eliminating duplication.
+  - Can be inserted “higher” in the IO path.
+  - Configurable caching parameters (memory vs performance)
+  - Will be employed to address Azure performance and throttling issues.
+  - Other disk modules converted over time.
+- User Interface
+  - Settings moved to top right navigation header
+  - C3 Charts fully implemented - chart interaction coming soon!
+  - Tagging for Ansible Tower job templates
+  - Live Search added to bootstrap selects
+  - Self Service UI Order History with detail
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+43+Ending+July+11%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Generic Object: Model refactoring/cleanup, use PostgreSQL jsonb column
+  - Null Coalescing Operator
+    - Multiple String values separated by “||”
+    - Evaluated on new attribute data type “Null Coalescing”
+    - Order dependent, left to right evaluation
+    - First non-blank value is used
+    - Skip and warn about missing objects
+- Performance
+  - Capacity and Utlization improvements included reduced number of SQL queries and number of objects
+  - Improved tag processing for Alert Profiles
+  - UI Performance: specific pages targeted resulted in up to 98% reduction in rendering for those pages
+- Platform: PostgreSQL upgraded to 9.5 needed for HA feature
+- Providers:
+  - Pluggability: Began extraction of Amazon Provider into separate repository
+  - Hawkular
+      - Upgraded to hawkular gem version 2.2.1
+      - Refactor infrastructure for easier configuration
+  - Ansible: Automate method updated to pass JobTemplate “Extra Variables” defined in the Provision Task
+- REST API: Refactoring
```

Comment:
If anything has been committed, it should be mentioned, unless it's inconsequential to a user (most refactorings don't actually change functionality, so probably should *not* be mentioned).  If it's all in-progress and not actually committed, it should not be mentioned at all.